### PR TITLE
Show connection id beside code catalyst dropdown

### PR DESCRIPTION
**Pull request by LarryWu12**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## Description
Connection id is now shown beside CodeCatalyst dropdown

Screenshot 2024-04-16 at 2 09 24 PM

## Checklist
- [X] My code follows the code style of this project
- [X] I have added tests to cover my changes
- [X] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/devToolsTab/nodes/CawsRootNode.kt [hunk 1]
```diff
@@ -45,16 +45,21 @@ class CawsRootNode(private val nodeProject: Project) : AbstractTreeNode<String>(
             is ActiveConnection.ValidBearer -> {
                 when (connection.connectionType) {
                     ActiveConnectionType.BUILDER_ID -> message("caws.connected.builder_id")
-                    else -> message("caws.connected.identity_center")
+                    else -> message(
+                        "caws.connected.identity_center",
+                        connection.activeConnectionBearer?.let { "(${it.id})" } ?: ""
```
Comment (manodnyab) [inline plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/devToolsTab/nodes/CawsRootNode.kt:50]: We can probably trim the sso:session text and just have the session name

Comment (LarryWu12) [reply plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/devToolsTab/nodes/CawsRootNode.kt:50]: I'm not sure all ids start with sso:session. Trimming just for this seems ad-hoc.

Comment (LarryWu12) [reply plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/devToolsTab/nodes/CawsRootNode.kt:50]: Ah it looks it's added in https://github.com/aws/aws-toolkit-jetbrains/blob/649c45dffffcb8e657d629b049e3035b0602bd4a/plugins/toolkit/core/src/software/aws/toolkits/core/credentials/ToolkitCredentialsProvider.kt#L140

**Comment by LarryWu12** [comment]
Screenshot 2024-04-18 at 10 18 26 AM

**Review by manodnyab — APPROVED** [review]

**Pull request merged**